bundles/core/src/test/resources/teaser/exporter-teaser9.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Teaser",
+  "description": "Description",
+  "linkURL": "http://www.adobe.com",
+  "withCTA": true,
+  "hideImageLink": false,
+  "hideTitleLink": false,
+  "ctas": [
+    {
+      "title": "Adobe",
+      "path": "http://www.adobe.com"
```

Comment:
seeing a URL at a path key seems strange here.

Reply:
I made the ctas/actions a list of `ListItem`s for POC of the mockups. Perhaps we should revisit this and use a different interface.